app/lib/hints.js

```diff
@@ -0,0 +1,83 @@
+module.exports = Base => class extends Base {
```

Comment:
Challenge. 
Do you think you could figure out how to get this in the database?
Also when you pull it out of the database I believe you can query 1 random one.

Reply:
adding issue for this